packages/youtube_player_flutter/lib/src/player/raw_youtube_player.dart

```diff
@@ -45,6 +45,7 @@ class _RawYoutubePlayerState extends State<RawYoutubePlayer>
   @override
   void dispose() {
     WidgetsBinding.instance.removeObserver(this);
+    controller?.value.webViewController?.dispose();
```

Comment:
It should not be disposed here as the controller is not initialized in this class.